queries.sql

```diff
@@ -14,3 +14,82 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
 SELECT * from animals WHERE name <> 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- query and update animals tables
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT before_weight_update;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO before_weight_update;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT species, MIN(weight_kg), MAX(weight_kg)
+FROM animals
+GROUP BY species;
+select neutered,sum(escape_attempts) as sum from animals GROUP BY neutered;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
+GROUP BY species;
+
+
+-- what animal belong to melody pond?
+select full_name, name from owners 
+join animals on animals.owners_id = owners.id where owners.full_name = 'Melody Pond';
+
```

Comment:
- [ ] @abenaTenkorang great work implementing the changes requested, however as the previous reviewer noted you have not updated the query screenshot in the pull request description, kindly consider updating it
![image](https://user-images.githubusercontent.com/61408860/236406948-b7fcf08f-8113-4335-b7a6-773d905eef43.png)